docs/_posts/2017-06-13-react-v15.6.0.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "React v15.6.0"
+author: flarnie
+---
+
+Today we are releasing React 15.6.0. As we prepare for React 16.0, we have been fixing and cleaning up many things. This release continues to pave the way.
+
+## Improving Inputs
+
+In React 15.6.0 the `onChange` event for inputs is a little bit more reliable and handles more edge cases, including the following:
+
+* not firing when radio button is clicked but not changed ([issue 1471](https://github.com/facebook/react/issues/1471))
+* changing an input of type 'range' with the arrow keys ([issue 554](https://github.com/facebook/react/issues/554))
+* pasting text into a text area in IE11 ([issue 7211](https://github.com/facebook/react/issues/7211))
+* auto-fill in IE11 ([issue 6614](https://github.com/facebook/react/issues/6614))
+* clearing input with 'x' button or right-click 'delete' in IE11 ([issue 6822](https://github.com/facebook/react/issues/6822))
+* not firing when characters are present in the input on render in IE11 ([issue 2185](https://github.com/facebook/react/issues/2185))
+
+Thanks to [Jason Quense](https://github.com/jquense) and everyone who helped out on those issues and PRs.
+
+## Downgrading Deprecation Warnings
```

Comment:
Agreed - not sure about 'Softening' either but will chat with folks and see what we come up with.